teuthology/suite/run.py

```diff
@@ -614,6 +622,9 @@ def schedule_suite(self):
         if jobs_to_schedule:
             self.write_rerun_memo()
 
+        if jobs_to_schedule and not self.args.skip:
```

Comment:
Maybe add a comment why this check should go exactly here?

Reply:
The priority should not be checked if `--priority` option is not used.

Reply:
Right, I have modified the condition.